Modules/_cursesmodule.c

```diff
@@ -1736,6 +1736,54 @@ _curses_window_inch_impl(PyCursesWindowObject *self, int group_right_1,
     return rtn;
 }
 
+#ifdef HAVE_NCURSESW
+/*[clinic input]
+_curses.window.in_wch
+
+    [
+    y: int
+        Starting Y-coordinate.
+    x: int
+        Starting X-coordinate.
+    ]
+    /
+
+Retrieve the wide character at the gven position in the window.
```

Comment:
```suggestion
Retrieve the Unicode character at the given position in the window.
```

Reply:
Ah, wait, should we say wide character or Unicode character here (it's a `wchar_t` but we use it to indicate "Unicode" range so...)